### PR TITLE
Fix several image tests.

### PR DESCRIPTION
**Pull request by zmarano**
/assign @gaohannk

### CODE REVIEW

File: imagetest/test_suites/disk/disk_resize_test.go [hunk 1]
```diff
@@ -32,19 +35,28 @@ func TestMain(m *testing.M) {
...
 func TestDiskResize(t *testing.T) {
-	_, err := os.Stat(markerFile)
+	image, err := utils.GetMetadata("image")
+	if err != nil {
+		t.Fatalf("couldn't get image from metadata")
+	}
+	switch {
+	case strings.Contains(image, "rhel-7-4-sap"):
+		t.Skip("disk expansion not supported on RHEL 7.4")
+	default:
```
Comment (hopkiw) [inline imagetest/test_suites/disk/disk_resize_test.go:45]: code execution halts at t.Skip, so no reason to capture it in a switch statement like this.

File: imagetest/test_suites/image_boot/image_boot_test.go [hunk 2]
```diff
@@ -43,15 +46,25 @@ func TestGuestReboot(t *testing.T) {
...
+		t.Fatalf("couldn't get image from metadata")
 	}
-	// https://www.kernel.org/doc/Documentation/ABI/stable/sysfs-firmware-efi-vars
-	if data[0] != 1 {
-		t.Fatal("secure boot is not enabled as expected")
+
+	switch {
+	case strings.Contains(image, "debian-9"):
+		t.Skip("secure boot is not supported on Debian 9")
+	default:
```
Comment (hopkiw) [inline imagetest/test_suites/image_boot/image_boot_test.go:57]: same feedback, no reason to use switch statements like this.

**Review by hopkiw — CHANGES_REQUESTED** [review]

File: imagetest/test_suites/disk/disk_resize_test.go (hunk 1 above)
Comment (zmarano) [reply imagetest/test_suites/disk/disk_resize_test.go]: Yeah that makes sense, fixed.

File: imagetest/test_suites/image_boot/image_boot_test.go (hunk 2 above)
Comment (zmarano) [reply imagetest/test_suites/image_boot/image_boot_test.go]: Ditto

File: imagetest/test_suites/security/image_security_test.go [hunk 3]
```diff
@@ -272,7 +272,7 @@ func verifyAutomaticUpdate(image string) error {
 		return err
 	}
 	switch {
-	case strings.Contains(image, "debian-9"):
+	case strings.Contains(image, "debian"):
```
Comment (gaohannk) [inline imagetest/test_suites/security/image_security_test.go:275]: Does this also valid for debian-10?

File: imagetest/test_suites/image_validation/license_test.go [hunk 4]
```diff
@@ -117,6 +118,7 @@ var licenses = []string{
 	`Open Market permits you to use, copy, modify, distribute, and license this Software and the Documentation for any purpose, provided that existing copyright notices are retained in all copies and that this notice is included verbatim in any distributions. No written agreement, license, or royalty fee is required for any of the authorized uses.`,
 	`This software is made available under the terms of *either* of the licenses found in LICENSE.APACHE or LICENSE.BSD. Contributions to cryptography are made under the terms of *both* these licenses.`,
 	`This library.*is public domain software`,
+	`This Font Software is licensed under the SIL Open Font License`,
```
Comment (hopkiw) [inline imagetest/test_suites/image_validation/license_test.go:121]: maybe a nitpick, but we usually cover these files with EITHER a license name OR a license text check; you've added both but i think one will suffice. usually we prefer license name, i think

File: imagetest/test_suites/security/image_security_test.go (hunk 3 above)
Comment (zmarano) [reply imagetest/test_suites/security/image_security_test.go:275]: Yup- at least in my test cases it was valid for all debian images (9-11)

**Pull request merged**